js/game.js

```diff
@@ -2,6 +2,12 @@ const MAX_DICE = 5;
 const MAX_REROLLS = 2;
 const SUBGAME_DICE = 3;
 
+let Bets = {
+    PASS: -1,
+    FOLD: -2,
+    CHECK: -3
+};
```

Comment:
I assume you're using negative constants to avoid inadvertent clashes ?

Reply:
I am abusing the fact that a bet can either be positive or any of that special values. Making them negative avoids that clash and fits in the same type.